Corona-Warn-App/src/main/res/values-de/contact_diary_strings.xml

```diff
@@ -12,7 +12,7 @@
     <string name="contact_diary_person_list_no_items_title">"Noch keine Personen vorhanden"</string>
     <string name="contact_diary_person_list_no_items_subtitle">"Legen Sie eine Person an und fügen Sie sie in Ihrem Kontakt-Tagebuch hinzu."</string>
     <string name="contact_diary_add_person_title">"Person"</string>
-    <string name="contact_diary_add_person_text_input_name_hint">"Vorname, Nachname"</string>
+    <string name="contact_diary_add_person_text_input_name_hint">"Name"</string>
```

Comment:
Wasn't it agreed that it should be `Vorname, Nachname`?

Reply:
Hi @harambasicluka , checked with Martin Hermes. We believe that 'Name' was agreed upon. Also - this seems to be current implementation in iOS.